openquake/mbt/tools/faults.py

```diff
@@ -93,10 +94,14 @@ def rates_for_double_truncated_mfd(area, slip_rate, m_low, m_upp, b_gr,
         Slip-rate
         float [mm/tr]
     :parameter m_low:
-        Lower magnitude
+        Reference magnitude [No m_min!, m_low <= m_min]
```

Comment:
Now this is m_low before we called it M_ref

Reply:
yep, here m_low is m_ref.

Reply:
I suggest to call it clipping magnitude `m_cli` everywhere (see proposed standard below).

Reply:
ok, it is fine with me